ruff.toml

```diff
@@ -10,18 +10,16 @@ lint.ignore = [
     "ANN205",   # Missing return type annotation for staticmethod
     "BLE001",   # Do not catch blind exception: `Exception`
     "C901",     # is too complex
-    "COM812",   # Trailing comma missing
-    "D203",     # 1 blank line required before class docstring
-    "D213",     # Multi-line docstring summary should start at the second line
+    "COM812",   # Trailing comma missing (conflicts with formatter)
+    "D203",     # 1 blank line required before class docstring (conflicts with `no-blank-line-before-class` (D211))
+    "D213",     # Multi-line docstring summary should start at the second line (conflicts with multi-line-summary-first-line` (D212))
```

Comment:
I added comments for the above so we know we can't disable them as they conflict with another rule